### PR TITLE
Add example for advanced video docking

### PR DESCRIPTION
**Pull request by alanorozco**

### CODE REVIEW

File: src/30_Advanced/Advanced_Video_Docking.html [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+      margin: 0;
+      padding-bottom: 20px;
+    }
+  </style>
+</head>
+
+<body>
+  <!-- ## Define animations -->
+  <!-- To slide down the content below the docking slot, the vertical offset is
+  changed into a 20px positive offset to create padding below the slot. -->
```
Comment (sebastianbenz) [inline src/30_Advanced/Advanced_Video_Docking.html]: nit: format `20px`

File: src/30_Advanced/Advanced_Video_Docking.html [hunk 2]
```diff
@@ -0,0 +1,223 @@
+<!---{
+  "hidePreview": true,
+  "experiments": ["video-dock"]
+}--->
```
Comment (sebastianbenz) [inline src/30_Advanced/Advanced_Video_Docking.html]: Can you please add a `introduction` section that shortly explains what this samples is about

File: src/30_Advanced/Advanced_Video_Docking.html [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+          }
+        ]
+      }
+    </script>
+  </amp-animation>
+  <header>My Header</header>
+  <main>
+    <div class="content">
+      <!-- ## Include your video -->
+      <!-- In your `<amp-video>` tag, you should include two attributes:
```
Comment (sebastianbenz) [inline src/30_Advanced/Advanced_Video_Docking.html]: Can you link `amp-video` to the reference doc.

File: src/30_Advanced/Advanced_Video_Docking.html (hunk 1 above)
Comment (alanorozco) [reply src/30_Advanced/Advanced_Video_Docking.html]: Done. Also formatted `-16.85vw`.

File: src/30_Advanced/Advanced_Video_Docking.html (hunk 2 above)
Comment (alanorozco) [reply src/30_Advanced/Advanced_Video_Docking.html]: Done.

File: src/30_Advanced/Advanced_Video_Docking.html (hunk 3 above)
Comment (alanorozco) [reply src/30_Advanced/Advanced_Video_Docking.html]: Done, the same with `amp-animation`.

File: src/30_Advanced/Advanced_Video_Docking.html [hunk 4]
```diff
@@ -2,16 +2,30 @@
   "hidePreview": true,
   "experiments": ["video-dock"]
```
Comment (sebastianbenz) [inline src/30_Advanced/Advanced_Video_Docking.html]: One thing I forgot: can you enable preview mode by adding: `"preview": "default"`.  Maybe also add a link to the preview in the introduction:

```
You can see the sample in action [here](preview).
```

Comment (alanorozco) [reply src/30_Advanced/Advanced_Video_Docking.html]: Preview template breaks the demo:

screen shot 2018-09-27 at 2 39 44 pm

Is there any way to force it to full-width?

Comment (sebastianbenz) [reply src/30_Advanced/Advanced_Video_Docking.html]: add:

```
main {
   width: 100%;
}
```

Preview is on my to fix list for amp.dev. 

Another thing, it looks a bit broken on mobile:

screenshot 2018-09-27 at 15 12 36

**Pull request merged**